.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+
+**Error Message**
+If applicable add error message to help explain your problem or console output enclose in triple back ticks eg,
+
+
+```console
```

Comment:
This will need to be enclosed in pre tags or it will process it and render as console output in this template. We should show how to do that here instead. I.e. `<pre></pre>`